looprpc/client.proto

```diff
@@ -190,6 +190,12 @@ message LoopOutRequest {
     lower total swap fee.
     */
     uint64 swap_publication_deadline = 10;
+
+    /*
+    An optional label for this swap. This field is limited to 500 characters
+    and may not be one of the reserved values in loop/labels Reserved list.
```

Comment:
We could also mention that it cannot start with the `[Reserved]` prefix.

Reply:
Ah yes indeed, this is leftover from a previous version.